### PR TITLE
Highlighting searched entities on map

### PR DESCRIPTION
**Pull request by borisvida**
1. change `selectedLocations` to an array, so more of them could be selected at once
2. show selected locations for all zoom levels
3. after searching throught `EntitySearchAutocomplete`, push all found entities to `selectedLocations` and zoom the map, so all of them are visible

### CODE REVIEW

File: client/src/components/Public/EntitySearchResult/EntitySearchResult.js [hunk 1]
```diff
@@ -1,43 +1,84 @@
...
+  componentDidUpdate(oldProps) {
+    if (this.props.searchFor !== oldProps.searchFor) {
+      this.highlightLocations()
+    }
+  }
+
+  highlightLocations = () => {
+    const {makeLocationsSelected, entityDetails, zoomToLocation} = this.props
+    const locations = uniqWith(
+      Object.values(entityDetails).map((detail) => pick(detail, ['lat', 'lng', 'address_id'])),
```
Comment (mirecmrozek) [inline client/src/components/Public/EntitySearchResult/EntitySearchResult.js]: No need to use Object.values if you use map from lodash.

File: client/src/selectors/index.js [hunk 2]
```diff
@@ -195,12 +197,38 @@ const createLabels = (mapOptions: MapOptions): Array<MapCluster> => {
...
+
 export const clustersSelector = createSelector(
   mapOptionsSelector,
   addressesSelector,
   useLabelsSelector,
-  (mapOptions, addresses, useLabels) =>
-    useLabels ? createLabels(mapOptions) : createClusters(mapOptions, addresses)
+  selectedLocationsSelector,
+  zoomSelector,
+  (mapOptions, addresses, useLabels, selectedLocations, zoom) => {
```
Comment (mirecmrozek) [inline client/src/selectors/index.js]: Zoom seems unneeded here.

File: client/src/components/Public/EntitySearchResult/EntitySearchResult.js (hunk 1 above)
Comment (borisvida) [reply client/src/components/Public/EntitySearchResult/EntitySearchResult.js]: True, but I prefer using native `map` for performance reasons :slightly_smiling_face:

File: client/src/selectors/index.js (hunk 2 above)
Comment (borisvida) [reply client/src/selectors/index.js]: Yeah, thanks :+1:

**Pull request merged**